doc/source/examples/incompressible-flow/2d-flow-around-cylinder/2d-flow-around-cylinder.rst

```diff
@@ -0,0 +1,95 @@
+======================================
+2D Flow around a cylinder
+======================================
+
+In this example, a flow is passing across a fixed cylinder. The velocity profile of the flow is simulated. The parameter file used is ``cylinder.prm``. This example can be solved using the ``gls_navier_stokes_2d`` solver.
+
+The following schematic describes the simulation.
+
+.. image:: images/geometry_bc.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: geometry_and_bc
+
+
+* bc = 0 (No slip boundary condition)
+* bc = 1 (u = 1; flow in the x-direction)
+* bc = 2 (Slip boundary condition)
+
+Initial conditions
+------------------
+The initial condition implemented describes the movement of the flow in the x-direction.
+
+.. code-block:: text
+
+    subsection initial conditions
+        set type = nodal
+        subsection uvwp
+            set Function expression = 1; 0; 0
+        end
+    end
+
+The ``subsection uvwp`` allows the description of a vector-valued function representing the velocity in the x-direction where the individual components of the velocity are separated by semicolons in the ``set Function expression``.
+
+Mesh
+----
+For the mesh, the ``gmsh`` option was chosen as which requires the input of a ``.msh`` file generated by `Gmsh <https://gmsh.info/#Download>`_.
+
+.. code-block:: text
+
+    subsection mesh
+        set type                 = gmsh
+        set file name            = cylinder_structured.msh
+    end
+
+The ``.msh`` file can be found in the same folder where the parameter file was obtained. The ``set file name`` includes the path of the file to be used, however since in this case both the parameter and the mesh files are located in the same folder, no path needs to be identified.
+
+Boundary conditions
+-------------------
+In this section, we mainly have 3 boundary conditions:
+
+* ``bc 0`` identifies the cylinder where we apply ``noslip`` boundary conditions on its walls. This leads to a velocity of 0 for the fluid directly in contact with the walls of the cylinder.
+* ``bc 1`` determines the flow of the fluid from the left wall. As mentioned before, the fluid is moving in the x-direction and therefore its boundary condition is defined with a function having a ``u`` velocity equals to 1.
+* ``bc2`` is applied at the top and bottom walls. This condition allows the simulation to be performed in a finite sized domain. In real life, the cylinder would be placed in a relatively infinite domain. Using ``slip`` condition, we assume that the fluid cannot go out in the normal direction, but that it can still flow from left to right without friction. Thus, the walls have no effect of the flow of the fluid.
+
```

Comment:
Add the explanation that there is no boundary condtion at the right and that this defines a no-stress boundary condition.